docs/_docs/integration-checks-reference.md

```diff
@@ -179,12 +179,7 @@ if the trigger key you specify doesn't exist.
 
 ## D006 - REST Hook Trigger Needs a Polling URL
 
-When users are setting up a hook-based (aka instant) Trigger, it's important to have
-a polling fallback. For example, imagine a Zap that triggers on a new Slack message.
-Without a polling URL, the test won't complete without the user sending an actual
-message in a Slack channel, which is disruptive. Instead, the test fetches a (real)
-recent message and uses it as the test result. The polling URL is only
-used for tests.
+For hook-based (aka instant) triggers, we require you to provide a Perform List URL so that users can pull a live sample in the Zap editor. This is called a Polling Sample and is something users need in order to be able to complete the setup of their Zaps and then turn them on.
```

Comment:
Hey @iscorer ! We just went through some discussion about this text in a couple of earlier PRs (https://github.com/zapier/visual-builder/pull/185, https://github.com/zapier/zapier/pull/48065, https://github.com/zapier/visual-builder/pull/196. Could I bother you to revise this a bit to keep the majority of the existing text? I'd suggest the following, but I'd love to see thoughts from @eliangcs as well.

> When users are setting up a hook-based (aka instant) Trigger, it's important to have
a polling fallback. For example, imagine a Zap that triggers on a new Slack message.
If testing relies on the sending of a webhook, the test won't complete without the user 
sending an actual message in a Slack channel, which is disruptive. 

> Instead, during testing, the Perform List (`performList`) operation fetches a 
(real) recent message using the provided URL for polling and uses it as the test result. 
The polling URL is only used for tests.

Reply:
I like @armchairlinguist's revision, which I think has more information from an end user's perspective. 

Also, remember to copy the update to [`z/z` repo](https://github.com/zapier/zapier/blob/695504d2447e04ea88130b3654b0481e405b0157/web/backend/zapier/app_platform/developer_cli/app_checker/checks/app_definition.py#L507-L512).

Reply:
Committed the suggested copy above. Re-requesting review. Will follow-up with changes to `z/z` once this PR has been merged.